apps/wakunode2/wakunode2.nim

```diff
@@ -498,6 +498,9 @@ proc startNode(node: WakuNode, conf: WakuNodeConf,
   if conf.keepAlive:
     node.startKeepalive()
 
+  # Maintain relay connections
+  node.startRelayConnectivityLoop()
```

Comment:
Does it make sense to start this when relay is not enabled?

Reply:
good catch. will fix.

Reply:
fixed